### ISSUE TITLE
Keyboard navigation does not work when slider is focused by clicking on the track

### ISSUE THREAD

**Issue by rkonstantinov** [telerik/kendo-react-inputs#15]
Also do we need to prevent the bubbling of the events. Currently, for example, hitting the home/end keys will bubble up to the parent elements.

**Comment by cosminnicula** [comment]
:+1: 

The Paciello Group seems to recommend  having only the `k-draghandle` anchor focusable http://files.paciellogroup.com/blogmisc/ARIA/slider - which is currently the case with the `Slider`. The problem is that when you click on the `SliderTrack` or `SliderTicks`, the focus doesn't jump to the `k-draghandle` anchor. Additionally, the `SliderButton`s would just respond to something like `enter` and `space`.

Maybe also it would be a good idea to extract this `keyBinding` common code in a `SliderUtil` function?..eventually this is just moving back and forth in a generic list.
